rotkehlchen/premium/premium.py

```diff
@@ -193,7 +196,11 @@ def sign(self, method: str, **kwargs: Any) -> tuple[hmac.HMAC, dict]:
             req['nonce'] = int(1000 * time.time())
         post_data = urlencode(req)
         hashable = post_data.encode()
-        message = urlpath.encode() + hashlib.sha256(hashable).digest()
+        if method == 'backup':
+            # nest uses hex for the backup
```

Comment:
why? Why this difference?

Reply:
Because python code whas concatenating the string `b'\xff\xfa'` with that format and the `\x` format for what I found is not trivial to reproduce.

Reply:
So it wasn't a purely binary format? How big is this `message`? It's just a hash right?

Perhaps explain that in a comment here please? Due to difference between this `\x` format and rust.

Reply:
no, it wasn't purely binary format. It was mixing normal string and then the `\x` format and then that as bytes